### PR TITLE
chore(wallet): upgrade to mui v5

### PR DESCRIPTION
**Pull request by samsiegart**

### CODE REVIEW

File: packages/dapp-wallet/ui/src/components/NavDrawer.jsx [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+import IconButton from '@mui/material/IconButton';
+import Menu from '@mui/icons-material/Menu';
+import Drawer from '@mui/material/Drawer';
 
 import NavMenu from './NavMenu';
 
 const NavDrawer = () => {
   const theme = useTheme();
   const [drawerOpened, setDrawerOpened] = useState(false);
-  const isMobile = useMediaQuery(theme.breakpoints.down('sm'));
```
Comment (michaelfig) [inline packages/dapp-wallet/ui/src/components/NavDrawer.jsx:12]: Did the breakpoints actually change, or is this unrelated to the upgrade?

File: packages/dapp-wallet/ui/src/components/NavMenu.jsx [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-import ListItem from '@material-ui/core/ListItem';
-import { ListItemText, ListItemIcon } from '@material-ui/core';
-import DashboardIcon from '@material-ui/icons/Dashboard';
-import Apps from '@material-ui/icons/Apps';
-import People from '@material-ui/icons/People';
-import AddCircle from '@material-ui/icons/AddCircle';
+import { makeStyles, useTheme } from '@mui/styles';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import { ListItemText, ListItemIcon } from '@mui/material';
```
Comment (michaelfig) [inline packages/dapp-wallet/ui/src/components/NavMenu.jsx]: I wonder if there's a more specific import for these `ListItem*` things than `'@mui/material'`.

**Review by michaelfig — APPROVED** [review]
Looks really good!  Just a few questions to consider... (oh and were the changes to the `color="xxx"` properties also required by the upgrade?)

File: packages/dapp-wallet/ui/src/components/NavDrawer.jsx (hunk 1 above)
Comment (samsiegart) [reply packages/dapp-wallet/ui/src/components/NavDrawer.jsx:12]: Yes, they changed the boundary of `breakpoints.down` to be non-inclusive, so you have to increase it by one. They have a script which handled most of these changes https://mui.com/guides/migration-v4/#run-codemods

**Comment by samsiegart** [comment]
> (oh and were the changes to the `color="xxx"` properties also required by the upgrade?)

Some were required, some were just related to the migration. I fixed the chip colors now that they're supported, and also the button color now defaults to "primary" so I:
- Removed `color="primary"` from the existing ones since it was redundant
- Added `color="cancel"` to fix the cancel buttons which were relying on the old default dark gray color

File: packages/dapp-wallet/ui/src/components/NavMenu.jsx (hunk 2 above)
Comment (samsiegart) [reply packages/dapp-wallet/ui/src/components/NavMenu.jsx]: Good catch, fixed

**Pull request merged**